### PR TITLE
[Django 2] Bytes to str

### PR DESCRIPTION
**Pull request by dannyroberts**
##### SUMMARY
In Django 2, functions that used to accept either `bytes` or unicode `str` now simply call `str` on anything that's passed in, changing the bytes to string conversion from `b'bytes' => 'bytes'` to `b'bytes' => "b'bytes'"`. This includes template variables and db field parameters.

### CODE REVIEW

File: corehq/ex-submodules/couchforms/migrations/0006_unfinishedarchivestub_attempts.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.AddField(
-            model_name='unfinishedarchivestub',
+            model_name='unfinishedarchivestu',
```
Comment (biyeun) [inline corehq/ex-submodules/couchforms/migrations/0006_unfinishedarchivestub_attempts.py]: @dannyroberts was this intentional to leave the `b` off then end in this migration? Seems like a find and replace error...

File: corehq/ex-submodules/couchforms/migrations/0001_initial.py [hunk 2]
```diff
@@ -8,7 +8,7 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.CreateModel(
-            name='UnfinishedSubmissionStub',
+            name='UnfinishedSubmissionStu',
```
Comment (biyeun) [inline corehq/ex-submodules/couchforms/migrations/0001_initial.py]: find & replace error

File: corehq/ex-submodules/couchforms/migrations/0002_auto_20170802_1505.py [hunk 3]
```diff
@@ -11,12 +11,12 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.AddField(
-            model_name='unfinishedsubmissionstub',
+            model_name='unfinishedsubmissionstu',
```
Comment (biyeun) [inline corehq/ex-submodules/couchforms/migrations/0002_auto_20170802_1505.py]: find & replace error

File: corehq/ex-submodules/couchforms/migrations/0002_auto_20170802_1505.py [hunk 4]
```diff
@@ -11,12 +11,12 @@ class Migration(migrations.Migration):
...
             name='attempts',
             field=models.IntegerField(default=0),
         ),
         migrations.AddField(
-            model_name='unfinishedsubmissionstub',
+            model_name='unfinishedsubmissionstu',
```
Comment (biyeun) [inline corehq/ex-submodules/couchforms/migrations/0002_auto_20170802_1505.py]: find & replace error

File: corehq/ex-submodules/couchforms/migrations/0003_auto_20170808_1212.py [hunk 5]
```diff
@@ -13,7 +13,7 @@ class Migration(migrations.Migration):
 
     operations = [
         AlterFieldCreateIndexIfNotExists(
-            model_name='unfinishedsubmissionstub',
+            model_name='unfinishedsubmissionstu',
```
Comment (biyeun) [inline corehq/ex-submodules/couchforms/migrations/0003_auto_20170808_1212.py]: find & replace error

File: corehq/ex-submodules/couchforms/migrations/0004_unfinishedarchivestub.py [hunk 6]
```diff
@@ -11,7 +11,7 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.CreateModel(
-            name='UnfinishedArchiveStub',
+            name='UnfinishedArchiveStu',
```
Comment (biyeun) [inline corehq/ex-submodules/couchforms/migrations/0004_unfinishedarchivestub.py]: find & replace error

File: corehq/ex-submodules/couchforms/migrations/0005_index_cleanup.py [hunk 7]
```diff
@@ -4,12 +4,12 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('couchforms', '0004_unfinishedarchivestub'),
+        ('couchforms', '0004_unfinishedarchivestu'),
```
Comment (biyeun) [inline corehq/ex-submodules/couchforms/migrations/0005_index_cleanup.py]: find & replace error

File: corehq/ex-submodules/couchforms/migrations/0005_index_cleanup.py [hunk 8]
```diff
@@ -4,12 +4,12 @@
...
     ]
 
     operations = [
         migrations.AddIndex(
-            model_name='unfinishedsubmissionstub',
+            model_name='unfinishedsubmissionstu',
```
Comment (biyeun) [inline corehq/ex-submodules/couchforms/migrations/0005_index_cleanup.py]: find & replace error

File: corehq/ex-submodules/couchforms/migrations/0007_auto_20191210_2206.py [hunk 9]
```diff
@@ -13,7 +13,7 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.AlterField(
-            model_name='unfinishedarchivestub',
+            model_name='unfinishedarchivestu',
```
Comment (biyeun) [inline corehq/ex-submodules/couchforms/migrations/0007_auto_20191210_2206.py]: find & replace error

File: corehq/sql_accessors/migrations/0056_add_hashlib_functions.py [hunk 10]
```diff
@@ -19,8 +19,8 @@ class Migration(migrations.Migration):
         # Todo: Move this to testing harness, doesn't really belong here.
         # See https://github.com/dimagi/commcare-hq/pull/21627#pullrequestreview-149807976
         migrations.RunSQL(
-            'CREATE EXTENSION IF NOT EXISTS hashlib',
-            'DROP EXTENSION hashlib'
+            'CREATE EXTENSION IF NOT EXISTS hashli',
```
Comment (biyeun) [inline corehq/sql_accessors/migrations/0056_add_hashlib_functions.py]: find & replace error

File: corehq/sql_proxy_accessors/migrations/0001_initial.py [hunk 11]
```diff
@@ -34,8 +34,8 @@ class Migration(migrations.Migration):
             'DROP EXTENSION plproxy'
         ),
         migrations.RunSQL(
-            'CREATE EXTENSION IF NOT EXISTS hashlib',
-            'DROP EXTENSION hashlib'
+            'CREATE EXTENSION IF NOT EXISTS hashli',
```
Comment (biyeun) [inline corehq/sql_proxy_accessors/migrations/0001_initial.py]: find & replace error

File: corehq/sql_proxy_standby_accessors/migrations/0001_initial.py [hunk 12]
```diff
@@ -34,8 +34,8 @@ class Migration(migrations.Migration):
             'DROP EXTENSION plproxy'
         ),
         migrations.RunSQL(
-            'CREATE EXTENSION IF NOT EXISTS hashlib',
-            'DROP EXTENSION hashlib'
+            'CREATE EXTENSION IF NOT EXISTS hashli',
```
Comment (biyeun) [inline corehq/sql_proxy_standby_accessors/migrations/0001_initial.py]: find & replace error

File: custom/aaa/migrations/0001_squashed_0002_auto_20190213_2342.py [hunk 13]
```diff
@@ -71,7 +71,7 @@ class Migration(migrations.Migration):
                 ('child_health_case_id', models.TextField(primary_key=True, serialize=False)),
                 ('opened_on', models.DateField(help_text='child_health.opened_on')),
                 ('closed_on', models.DateField(help_text='child_health.opened_on', null=True)),
-                ('dob', models.DateField(null=True)),
+                ('do', models.DateField(null=True)),
```
Comment (biyeun) [inline custom/aaa/migrations/0001_squashed_0002_auto_20190213_2342.py]: find & replace error

File: custom/aaa/migrations/0001_squashed_0002_auto_20190213_2342.py [hunk 14]
```diff
@@ -97,7 +97,7 @@ class Migration(migrations.Migration):
                 ('opened_on', models.DateField()),
                 ('closed_on', models.DateField(null=True)),
                 ('pregnant_ranges', django.contrib.postgres.fields.ArrayField(base_field=django.contrib.postgres.fields.ranges.DateRangeField(), help_text='The ranges in which a ccs_record has been opened and the baby has not been born', null=True, size=None)),
-                ('dob', models.DateField(null=True)),
+                ('do', models.DateField(null=True)),
```
Comment (biyeun) [inline custom/aaa/migrations/0001_squashed_0002_auto_20190213_2342.py]: find & replace error

File: custom/icds_reports/migrations/0050_move_to_django.py [hunk 15]
```diff
@@ -453,7 +453,7 @@ class Migration(migrations.Migration):
                 ('valid_in_month', models.IntegerField(blank=True, null=True)),
                 ('valid_all_registered_in_month', models.IntegerField(blank=True, null=True)),
                 ('ebf_no_info_recorded', models.IntegerField(blank=True, null=True)),
-                ('dob', models.DateField(blank=True, null=True)),
+                ('do', models.DateField(blank=True, null=True)),
```
Comment (biyeun) [inline custom/icds_reports/migrations/0050_move_to_django.py]: find & replace error

File: custom/icds_reports/migrations/0068_ccsrecordmonthlyview_add_lactating.py [hunk 16]
```diff
@@ -6,7 +6,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('icds_reports', '0067_ccsrecordmonthly_dob'),
+        ('icds_reports', '0067_ccsrecordmonthly_do'),
```
Comment (biyeun) [inline custom/icds_reports/migrations/0068_ccsrecordmonthlyview_add_lactating.py]: find & replace error

File: custom/icds_reports/migrations/0112_aggawcdailyview_agglsmonthly_awwincentivereportmonthly_childhealthmonthlyview_servicedeliverymonthly.py [hunk 17]
```diff
@@ -137,7 +137,7 @@ class Migration(migrations.Migration):
                 ('state_id', models.TextField(blank=True, null=True)),
                 ('person_name', models.TextField(blank=True, null=True)),
                 ('mother_name', models.TextField(blank=True, null=True)),
-                ('dob', models.DateField(blank=True, null=True)),
+                ('do', models.DateField(blank=True, null=True)),
```
Comment (biyeun) [inline custom/icds_reports/migrations/0112_aggawcdailyview_agglsmonthly_awwincentivereportmonthly_childhealthmonthlyview_servicedeliverymonthly.py]: find & replace error

**Review by biyeun — CHANGES_REQUESTED** [review]

**Comment by dannyroberts** [comment]
Gah, thank you so much @biyeun. Definitely a rookie find and replace error

**Review by biyeun — APPROVED** [review]
@dannyroberts no problem! looks good now

**Comment by dannyroberts** [comment]
Travis says ✅ when you click through

**Pull request merged**